src/dawn/pipeline_info.h

```diff
@@ -50,6 +50,7 @@ struct RenderPipelineInfo {
   // The number of data bytes in the framebuffer host-side buffer.
   uint32_t fb_size = 0;
   bool fb_is_mapped = false;
+  const void* fb_data = nullptr;
```

Comment:
Every time fb_is_mapped is set, the fb_data member should be set.  For example where the buffer is unmapped.

Reply:
I had another thought.   I want fb_data to be nullptr if and only if the buffer is *unmapped*.   But that makes fb_is_mapped redundant because it encodes the same information.
So: remove the fb_is_mapped variable entirely, and only set and check fb_data.